src/Button/Button.less

```diff
@@ -22,6 +22,7 @@ a, button {
   &.Button {
     .border--s(transparent);
     .borderRadius--top--m;
+    .borderRadius--bottom--0;
```

Comment:
could you also all a `.borderRadius--0` to the `.button--reset()` mixin? that will also fix other buttons like the ones in the left nav: https://github.com/Clever/components/blob/master/src/less/resets.less#L4

we'll need the same fix on the segmented control as well: https://github.com/Clever/components/blob/master/src/SegmentedControl/SegmentedControl.less#L7
(it should probably eventually be updated to use `.button--reset()` as well)

Reply:
Smart... will add that to this PR